Assignment/Program.cs

```diff
@@ -1,7 +1,15 @@
-﻿namespace Assignment {
-   internal class Program {
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello world!");
-      }
+﻿// Find factorial for the given number.
+while (true) {
+   Console.Write ("Give number to find factorial : ");
+   if (int.TryParse (Console.ReadLine (), out int input) && (input > 0)) {

```

Comment:
Can you restrict the inputs apart from between 1 and 25. (Type int can handle upto that level only)